File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/articles_edited/view/ArticlesEditedView.java
package org.wikiedufoundation.wikiedudashboard.course_detail.articles_edited.view;

import org.wikiedufoundation.wikiedudashboard.course_detail.articles_edited.data.ArticlesEdited;

import org.wikiedufoundation.wikiedudashboard.helper.Progressive;
import org.wikiedufoundation.wikiedudashboard.helper.Toaster;

public interface ArticlesEditedView extends Progressive, Toaster {
    void setData(ArticlesEdited data);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/articles_edited/view/ArticlesEditedView.kt
package org.wikiedufoundation.wikiedudashboard.course_detail.articles_edited.view

import org.wikiedufoundation.wikiedudashboard.course_detail.articles_edited.data.ArticlesEdited

import org.wikiedufoundation.wikiedudashboard.helper.Progressive
import org.wikiedufoundation.wikiedudashboard.helper.Toaster

interface ArticlesEditedView : Progressive, Toaster {
    fun setData(data: ArticlesEdited)
}
